src/Blueprint.php

```diff
@@ -2,16 +2,28 @@
 
 namespace Blueprint;
 
-
-use Blueprint\Contracts\Generator;
 use Blueprint\Contracts\Lexer;
 use Symfony\Component\Yaml\Yaml;
+use Blueprint\Contracts\Generator;
 
 class Blueprint
 {
     private $lexers = [];
     private $generators = [];
 
+    public function boot()
+    {
+        collect(config('blueprint.lexers'))->each(function ($lexer) {
+            $this->registerLexer(new $lexer());
```

Comment:
```suggestion
            $this->registerLexer($this->laravel->make($lexer));
```

This resolves the lexer from the container, allowing the creator of third-party lexers to inject dependences into its constructor.